HTTPServerProject.Tests/HTTPServerProxyTests/ResponseTests.cs

```diff
@@ -0,0 +1,224 @@
+using HTTPServerReadTests.Streams;
+using HTTPServerProxy.Response;
+using HTTPServerWriteTests.Streams;
+
+namespace HTTPServerProxyTests.Response;
+
+public class ProxyResponseTests
+{
+    [Fact]
+    public void GetTodoPOSTValidResponseTest()
+    {
+        var expectedStatusLine = "HTTP/1.1 201 Created";
+        var proxyStream = new List<string>() 
+		{
+        "HTTP/1.1 200 OK", 
+		"Content-Type: application/json", 
+		"Content-Length: 20", 
+		"", 
+		"{ 'task': 'First Item' }" 
+		};
+        var stream = new List<string>();
+        var reader = new TestReadStreams(proxyStream);
+        var httpType = "POST";
+        var httpPath = "todo";
+        var writer = new TestWriteStreams(stream);
+
+        var proxyResponse = new ProxyResponse(reader: reader, writer: writer, path: httpPath, type: httpType);
+        proxyResponse.GetResponse();
+        var actualStatusLine = stream[0];
+        Assert.Equal(expectedStatusLine, actualStatusLine);
```

Comment:
Moving expectedStatusLine down here as well is also a bit easier to read.

Reply:
Agreed.